### ISSUE TITLE
Fail to start docker container the second time.

### ISSUE THREAD

**Issue by paranoid945** [DataDog/datadog-agent#1565]
**Output of the info page (if this is a bug)**
```
[cont-init.d] 01-check-apikey.sh: executing... 
[cont-init.d] 01-check-apikey.sh: exited 0.
[cont-init.d] 10-dogstatsd-socket.sh: executing... 
[cont-init.d] 10-dogstatsd-socket.sh: exited 0.
[cont-init.d] 50-ecs.sh: executing... 
[cont-init.d] 50-ecs.sh: exited 0.
[cont-init.d] 50-kubernetes.sh: executing... 
[cont-init.d] 50-kubernetes.sh: exited 0.
[cont-init.d] 50-mesos.sh: executing... 
[cont-init.d] 50-mesos.sh: exited 0.
[cont-init.d] 51-docker.sh: executing... 
mv: cannot stat '/etc/datadog-agent/conf.d/docker.d/conf.yaml.example': No such file or directory
[cont-init.d] 51-docker.sh: exited 1.

```

**Describe what happened:**
I can successfully start the docker container the first time, but I can not start it successful the second time.

**Describe what you expected:**
I should always start the container sucessfully.

**Steps to reproduce the issue:**
start it, stop it, start it again.

**Additional environment details (Operating System, Cloud provider, etc):**
docker-compose.yml
```
version: '2'
services:
  datadog-agent:
    image: datadog/agent:6.1.2
    container_name: datadog
    network_mode: "host"
    hostname: s1.local
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock:ro
      - /proc/:/host/proc/:ro
      - /sys/fs/cgroup/:/host/sys/fs/cgroup:ro
      - "./conf.d/:/conf.d/:ro"
    restart: always
    environment:
      - DD_API_KEY=xxxxxxxxxxxxxxxxxxxxxxxx
      - DD_LOG_LEVEL=WARNING
```
// start command:
docker-compose -f docker-compose.yml up -d && docker-compose -f docker-compose.yml logs -f -t
// stop the container:
docker stop datadog
// start it again:
docker start datadog

**Investigation**
The problem could be this script:
/etc/cont-init.d/51-docker.sh
```
#!/bin/bash
#
# We enable docker if either:
#   - we detect the DOCKER_HOST envvar, overriding the default socket location
#     (in that case, we trust the user wants docker integration and don't check 
existence)
#   - we find the docker socket at it's default location

if [[ -z "${DOCKER_HOST}" && ! -e /var/run/docker.sock ]]; then
    exit 0
fi


# Set a config for vanilla Docker if no orchestrator was detected
# by the 50-* scripts
# Don't override /etc/datadog-agent/datadog.yaml if it exists
if [[ ! -e /etc/datadog-agent/datadog.yaml ]]; then
    ln -s  /etc/datadog-agent/datadog-docker.yaml \
           /etc/datadog-agent/datadog.yaml
fi

# Enable the docker corecheck
mv /etc/datadog-agent/conf.d/docker.d/conf.yaml.example \
   /etc/datadog-agent/conf.d/docker.d/conf.yaml.default
```

The first time it successflly change the name from conf.yaml.example to conf.yaml.default, but at the second time it can not find the file name "conf.yaml.default" because it is already been changed, so the script fail to execute and the container fail to start.

**Comment by mfpierre** [comment]
Hi @paranoid945 thanks for the detailed report, this should be fixed by #1541 that will go out in the next release (6.2)

**Comment by paranoid945** [comment]
Thanks for your quick response, I switched back to 6.0.3, everything works fine.
